src/train.py

```diff
@@ -2,49 +2,73 @@
 from training_data import TrainingData
 from rasa_nlu.util import update_config
 import json
+import warnings
+import os
 
 
 def create_argparser():
     parser = argparse.ArgumentParser(description='train a custom language parser')
+
     # TODO add args for training only entity extractor or only intent
     parser.add_argument('-b', '--backend', default='keyword', choices=['mitie', 'spacy_sklearn', 'keyword'],
                         help='backend to use to interpret text (default: built in keyword matcher).')
     parser.add_argument('-p', '--path', default=None, help="path where model files will be saved")
     parser.add_argument('-d', '--data', default=None, help="file containing training data")
     parser.add_argument('-c', '--config', required=True, help="config file")
     parser.add_argument('-l', '--language', default='en', choices=['de', 'en'], help="model and data language")
+    parser.add_argument('-m', '--mitie_file', default='data/total_word_feature_extractor.dat',
+                        help='file with mitie total_word_feature_extractor')
     return parser
 
 
 def create_trainer(config):
     backend = config['backend'].lower()
     if backend == 'mitie':
         from trainers.mitie_trainer import MITIETrainer
-        return MITIETrainer(config['backends']['mitie'], config['language'])
+        return MITIETrainer(config['mitie_file'], config['language'])
     if backend == 'spacy_sklearn':
         from trainers.spacy_sklearn_trainer import SpacySklearnTrainer
-        return SpacySklearnTrainer(config['backends']['spacy_sklearn'], config['language'])
+        return SpacySklearnTrainer(config['language'])
     else:
         raise NotImplementedError("other backend trainers not implemented yet")
 
 
 def load_configuration(file_name):
-    return json.loads(open(file_name, 'rb').read())
+    config = {}
+    if (os.path.isfile(file_name)):
+        config = json.loads(open(file_name, 'rb').read())
+    else:
+        warnings.warn("could not find config file {0}, ignoring".format(file_name))
+    return config
+
+
+def create_persistor(config):
+    persistor = None
+    try:
+        from rasa_nlu.persistor import Persistor
+        persistor = Persistor(config['path'], config['aws_region'], config['bucket_name'])
+    except:
+        pass
```

Comment:
If that is the only thing that could go wrong, I would go for an exception free version as well